float64 rosTime
float32 estimatedTW
